test/jdk/com/sun/net/httpserver/HttpPrincipalTest.java

```diff
@@ -36,9 +36,13 @@
 public class HttpPrincipalTest {
 
     @Test
-    public void TestGetters() {
+    public void testGetters() {
         var principal = new HttpPrincipal("test", "123");
+
         assertEquals(principal.getUsername(), "test");
         assertEquals(principal.getRealm(), "123");
+        assertEquals(principal.getName(), "test:123");
```

Comment:
should be "123:test" here.

Reply:
Fixed in commit 2ad0c8d8a0d21713b6b3cdbd59306fd4d57bf4d7